### ISSUE TITLE
Failed to activate the atom-ctags package

### ISSUE THREAD

**Issue by kbthomp1** [yongkangchen/atom-ctags#188]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.11.6
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package 5.0.0

### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:1)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at /packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at Object.activate (/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:225:1
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:99:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:218:1
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:215:1)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:642:1)
```

### Commands

```
     -0:10.1.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.use-animations.theme-fortran-syntax.theme-atom-material-ui)
```

### Non-Core Packages

```
atom-ctags 5.0.0 
atom-debugger 0.1.6 
atom-html-preview 0.1.22 
atom-material-syntax 1.0.6 
atom-material-ui 1.3.10 
autoclose-html 0.23.0 
autocomplete-clang 0.11.3 
autocomplete-fortran 0.9.4 
autocomplete-java 1.2.5 
autocomplete-python 1.8.63 
build 0.67.0 
build-xcodebuild 0.2.0 
busy 0.7.0 
busy-signal 1.4.1 
erb-helper 1.1.0 
fortran-syntax 0.0.0 
git-plus 7.8.0 
git-time-machine 1.5.9 
goto-last-edit 0.3.2 
intentions 1.1.2 
language-autotools 2.0.1 
language-fortran 2.1.0 
language-groovy 0.6.0 
line-continuation-fortran 0.0.0 
linter 2.1.4 
linter-clang 4.1.1 
linter-gcc 0.7.1 
linter-gfortran 0.5.0 
linter-python 3.1.1 
linter-shellcheck 1.4.4 
linter-ui-default 1.2.4 
pigments 0.39.1 
python-tools 0.6.9 
remote-sync 4.1.5 
seti-classic 0.10.3 
seti-classic-syntax 0.5.0 
symbols-tree-view 0.14.0 
sync-settings 0.8.1 
vim-mode-plus 0.92.0 
vim-mode-plus-ex-mode 0.9.0 
```

**Comment by reasonMix** [comment]
so am i,please solve it

**Comment by s5s5** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 ia32
**Electron**: 1.3.15
**OS**: Unknown Windows version
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package 5.0.0


### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module 'C:\Users\superliu\AppData\Local\atom\app-1.17.0\resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module 'C:\Users\superliu\AppData\Local\atom\app-1.17.0\resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (~/AppData/Local/atom/app-1.17.0/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.17.0/resources/app/src/module-cache.js:354:1)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:40:43)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (~/AppData/Local/atom/app-1.17.0/resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at /packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (~/AppData/Local/atom/app-1.17.0/resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at Object.activate (/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:253:1)
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:225:1
    at Package.module.exports.Package.measure (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:99:1)
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:218:1
    at Package.module.exports.Package.activate (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:215:1)
    at PackageManager.module.exports.PackageManager.activatePackage (~/AppData/Local/atom/app-1.17.0/resources/app/src/package-manager.js:642:1)
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/package-manager.js:374:1
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/config.js:644:1
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.17.0/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.17.0/resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Config.module.exports.Config.emitChangeEvent (~/AppData/Local/atom/app-1.17.0/resources/app/src/config.js:835:1)
    at Config.module.exports.Config.setRawValue (~/AppData/Local/atom/app-1.17.0/resources/app/src/config.js:621:1)
    at Config.module.exports.Config.set (~/AppData/Local/atom/app-1.17.0/resources/app/src/config.js:236:1)
    at Config.module.exports.Config.removeAtKeyPath (~/AppData/Local/atom/app-1.17.0/resources/app/src/config.js:386:1)
    at Package.module.exports.Package.enable (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:84:1)
    at PackageManager.module.exports.PackageManager.enablePackage (~/AppData/Local/atom/app-1.17.0/resources/app/src/package-manager.js:185:1)
    at HTMLButtonElement.enablementButtonClickHandler (~/AppData/Local/atom/app-1.17.0/resources/app/node_modules/settings-view/lib/package-card.js:361:1)
```

### Commands

```
```

### Non-Core Packages

```
activate-power-mode 1.4.0 
atom-beautify 0.29.24 
atom-ctags 5.0.0 
atom-ternjs 0.18.3 
autocomplete-paths 1.0.5 
busy-signal 1.4.1 
emmet 2.4.3 
file-icons 2.1.6 
highlight-selected 0.13.1 
intentions 1.1.2 
linter 2.1.4 
linter-ui-default 1.5.4 
```

**Comment by RickyFan1015** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package 5.0.0


### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:1)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at /packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:1)
    at Object.activate (/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:225:1
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:99:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:218:1
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:215:1)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:642:1)
```

### Commands

```
```

### Non-Core Packages

```
activate-power-mode 1.4.0 
atom-ctags 5.0.0 
atom-material-syntax 1.0.6 
atom-material-ui 2.0.3 
change-case 0.6.5 
file-icons 2.1.6 
git-control 0.9.0 
highlight-selected 0.13.1 
language-cmake 0.3.0 
project-manager 3.3.4 
todo-show 1.11.0 
```

**Comment by zvit1** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 ia32
**Electron**: 1.3.15
**OS**: Unknown Windows version
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package 5.0.0


### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module 'C:\Users\Administrator\AppData\Local\atom\app-1.17.0\resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module 'C:\Users\Administrator\AppData\Local\atom\app-1.17.0\resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (~/AppData/Local/atom/app-1.17.0/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.17.0/resources/app/src/module-cache.js:354:1)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:40:43)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at /packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (~/AppData/Local/atom/app-1.17.0/resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at /packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at /packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:106:1)
    at Object.value [as .coffee] (~/AppData/Local/atom/app-1.17.0/resources/app/src/compile-cache.js:230:1)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (~/AppData/Local/atom/app-1.17.0/resources/app/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.17.0/resources/app/src/native-compile-cache.js:66:1)
    at Object.activate (/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:253:1)
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:225:1
    at Package.module.exports.Package.measure (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:99:1)
    at ~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:218:1
    at Package.module.exports.Package.activate (~/AppData/Local/atom/app-1.17.0/resources/app/src/package.js:215:1)
    at PackageManager.module.exports.PackageManager.activatePackage (~/AppData/Local/atom/app-1.17.0/resources/app/src/package-manager.js:642:1)
```

### Commands

```
```

### Non-Core Packages

```
atom-bootstrap3 1.2.12 
atom-ctags 5.0.0 
atom-live-server 2.1.2 
auto-indent 0.5.0 
busy-signal 1.4.1 
emmet 2.4.3 
highlight-selected 0.13.1 
intentions 1.1.2 
language-nunjucks 4.1.0 
language-vue 0.21.2 
linter 2.1.4 
linter-eslint 8.1.7 
linter-ui-default 1.2.2 
minimap 4.28.2 
Quick-JavaScript 2.2.0 
react 0.16.2 
react-snippets 0.7.2 
tortoise-svn 0.5.0 
```

**Comment by yongkangchen** [comment]
Fixed in atom-ctags v5.1.0

**Comment by louisrubet** [comment]
Sorry, not fixed in atom-ctags v5.1.0

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.10.0
**Electron Version**: 0.37.8
**System**: Ubuntu 16.10
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package, v5.1.0


### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module '/usr/share/atom/resources/app/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module '/usr/share/atom/resources/app/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:339:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/louis/.atom/packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at Object.<anonymous> (/home/louis/.atom/packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/louis/.atom/packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at Object.<anonymous> (/home/louis/.atom/packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.module.exports.activate (/home/louis/.atom/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:183:20)
    at /usr/share/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:92:15)
    at /usr/share/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:538:21)
    at /usr/share/atom/resources/app.asar/src/package-manager.js:342:19
    at /usr/share/atom/resources/app.asar/src/config.js:635:20
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at Config.module.exports.Config.emitChangeEvent (/usr/share/atom/resources/app.asar/src/config.js:827:29)
    at Config.module.exports.Config.setRawValue (/usr/share/atom/resources/app.asar/src/config.js:612:19)
    at Config.module.exports.Config.set (/usr/share/atom/resources/app.asar/src/config.js:227:14)
    at Config.module.exports.Config.removeAtKeyPath (/usr/share/atom/resources/app.asar/src/config.js:377:12)
    at Package.module.exports.Package.enable (/usr/share/atom/resources/app.asar/src/package.js:77:26)
    at PackageManager.module.exports.PackageManager.enablePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:169:14)
    at HTMLButtonElement.<anonymous> (/usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-card.js:343:27)
    at HTMLButtonElement.jQuery.event.dispatch (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLButtonElement.elemData.handle (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "symbols-view"
    ],
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  },
  "atom-ctags": {
    "autoBuildTagsWhenActive": true
  }
}
```

### Installed Packages

```coffee
# User
atom-ctags, v5.1.0 (inactive)
atom-meld, v0.3.3 (active)
github-atom-light-syntax, v0.3.2 (inactive)
hex, v0.6.2 (active)
platformio-ide-terminal, v2.4.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (active)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (active)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.241.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```
